### ISSUE TITLE
Subgroup limits aren

### ISSUE THREAD

**Issue by alan-baker** [gpuweb/gpuweb#4834]

**Comment by dneto0** [comment]
cross-link to subgroups issue #4306

**Comment by kainino0x** [comment]
Yes, I agree we should add properties (as a separate concept from limits) as soon as we need them.

**Comment by beaufortfrancois** [comment]
For info, webgpu.h has GPUAdapterProperties which were the previous names of GPUAdapterInfo: https://source.chromium.org/chromium/chromium/src/+/main:third_party/dawn/third_party/webgpu-headers/webgpu.h;l=1568;drc=8049c324dc7b3c09dc96ea04cb02860f272c8686
We may want to be careful about the naming to not confuse developers
Maybe GPUAdapterSettings or GPUAdapterAttributes ?

**Comment by kainino0x** [comment]
I like "Properties" because that matches the naming in Vulkan. The webgpu.h concern is temporary and I don't think it will cause significant problems.
